.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "sourceType": "module"
+    },
+    "rules": {
+        "indent": [
+            "error",
+            4
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        // "quotes": [
```

Comment:
eslint rule for spaces before first char in a comment
https://eslint.org/docs/rules/spaced-comment